storybook/src/admin-rte/MaxListLevel.tsx

```diff
@@ -9,13 +8,29 @@ const [useRteApi] = makeRteApi();
 
 export default {
     title: "@comet/admin-rte",
+    args: {
+        listLevelMax: 4,
+    },
+    argTypes: {
+        listLevelMax: {
+            name: "Max list level",
+            control: {
+                type: "number",
+                min: 1,
```

Comment:
Was this changed to `1` from `0` on purpose?

Reply:
No, thanks. Fixed: 5595c71e000968980c4c78b52c3b9d3307ec2869